crates/turbopack-ecmascript-plugins/src/transform/swc_ecma_transform_plugins.rs

```diff
@@ -29,21 +29,8 @@ pub struct SwcPluginModule(
 );
 
 impl SwcPluginModule {
-    pub fn new(plugin_name: &str, plugin_bytes: Vec<u8>) -> Self {
-        Self {
-            #[cfg(feature = "swc_ecma_transform_plugin")]
-            0: {
-                use swc_core::plugin_runner::plugin_module_bytes::{
-                    CompiledPluginModuleBytes, RawPluginModuleBytes,
-                };
-                CompiledPluginModuleBytes::from(RawPluginModuleBytes::new(
-                    plugin_name.to_string(),
-                    plugin_bytes,
-                ))
-            },
-            #[cfg(not(feature = "swc_ecma_transform_plugin"))]
-            0: (),
-        }
+    pub fn new(_plugin_name: &str, _plugin_bytes: Vec<u8>) -> Self {
+        Self(())
```

Comment:
what happened here? Is this a safe change?

Reply:
I don't think so